### PR TITLE
Google and Office 365 Docs

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

**Comment by jasonblais** [comment]
Config-setting updates will be submitted with #333

**Comment by lindy65** [comment]
+1

File: source/deployment/sso-google.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Under **Restrictions** > **Authorized redirect URIs**, enter `your-mattermost-url/signup/google/complete` (example: `http://localhost:8065/signup/google/complete`). Click **Create**.
+
+![google_4_web_app](../../source/images/google_4_web_app.PNG)
+
+6 - Copy the **Client ID** and **Client Secret**, which will be used to configure Google SSO for Mattermost.
+
+![google_5_client_id_secret](../../source/images/google_5_client_id_secret.PNG)
+
+7 - Finally, go to [Google+ API](https://console.developers.google.com/apis/api/plus/overview) and click **Enable** in the header. This might take a few minutes to propagate through Google's systems.
+8 - In **System Console > OAuth 2.0 > Select OAuth 2.0 service provider**, choose **Google Apps** as the service provider and enter **Client ID** and **Client Secret** from step 6 in their respective fields.
```
Comment (esethna) [inline source/deployment/sso-google.md]: Needs a line break or the number doesn't show on a new line

File: source/deployment/sso-google.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Google Single-Sign-On Setup (Enterprise)
+___
+
+Follow these steps to configure Mattermost to use Google as a single-sign-on (SSO) service for team creation, account creation and sign-in.
+
+1 - [Log in](https://accounts.google.com/login) to your Google account.
+2 - Go to [Google API Manager](https://console.developers.google.com), click **Credentials** in the left hand sidebar, click *Create a project** and enter "Mattermost - your-company-name" as the **Project Name**, then click **Create**.
+
+![google_1_credentials](../../source/images/google_1_credentials.PNG)
```
Comment (esethna) [inline source/deployment/sso-google.md]: Most of these images aren't rendering. Possibly due to the capitalization of the "PNG" here but not in the image file name?

**Comment by esethna** [comment]
@jasonblais added a few notes for fixes above^

**Comment by jasonblais** [comment]
Fixed @esethna

**Comment by esethna** [comment]
+1

**Pull request merged**